openshift-with-appstudio-test/e2e/jvm_build_service_test.go

```diff
@@ -45,15 +45,15 @@ func TestExampleRun(t *testing.T) {
 		debugAndFailTest(ta, fmt.Sprintf("file %s did not produce a task: %#v", mavenYamlPath, obj))
 	}
 	// override images if need be
-	analyserImage := os.Getenv("JVM_BUILD_SERVICE_ANALYZER_IMAGE")
+	analyserImage := os.Getenv("JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE")
 	if len(analyserImage) > 0 {
 		ta.Logf(fmt.Sprintf("PR analyzer image: %s", analyserImage))
-		for _, step := range ta.maven.Spec.Steps {
+		for i, step := range ta.maven.Spec.Steps {
 			if step.Name != "analyse-dependencies" {
 				continue
 			}
 			ta.Logf(fmt.Sprintf("Updating analyse-dependencies step with image %s", analyserImage))
-			step.Image = analyserImage
+			ta.maven.Spec.Steps[i].Image = analyserImage
```

Comment:
@gabemontero looks like we were not actually using the dependency analyser built in CI, but because it does not change much we did not pick up on it.

Reply:
ah another forgot how golang slices work mistake (either in coding or review of another's PR) :-/